Webhooks - Add more info in github actions payload example (#493)
wandb/docs: docs/guides/model_registry/automation.md

  }

  ```
:::note

> The `event_type` key in the automation payload must match the `types` field in the GitHub workflow YAML file.

Reviewer: ```suggestion
The `event_type` key in the webhook payload must match the `types` field in the GitHub workflow YAML file.
```

:::

>   Where template strings render depending on the event or model version the automation is configured for. `${event_type}` will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:

Reviewer: ```suggestion
  Where template strings render depends on the event or model version the automation is configured for. `${event_type}` will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:
```
@ngrayluna for some reason this sentence is troubling me - should it be depends?

  }

  ```
:::note
The `event_type` key in the webhook payload must match the `types` field in the GitHub workflow YAML file.
:::

>   Where template strings render depend on the event or model version the automation is configured for. `${event_type}` will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:

Reviewer: ```suggestion
The contents and positioning of the rendered template strings depends on the event or model version the automation is configured for. ${event_type} will render as either "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS". See below for an example mapping:
```
